.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: true
+language: node_js
+node_js:
+  - "8"
+
+branches:
+  only:
+    - /.*/
+
+before_install:
+  - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+  - echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+
+install:
+  - sudo apt update && sudo apt install -y yarn
+  - /usr/bin/yarn install --production=false
+  # - npm run test
+  - npm run flow
+
+script:
+  - npm run jsdoc
```

Comment:
It would be better to use one utility: npm or yarn. This command could be replaced with yarn jsdoc